app/javascript/article-form/elements/imageManagement.jsx

```diff
@@ -120,7 +128,12 @@ export default class ImageManagement extends Component {
     } else {
       mainImageArea = (
         <div>
-          <input type="file" onChange={this.handleMainImageUpload} />
+          <input
+            type="file"
+            onChange={this.handleMainImageUpload}
+            accept="image/*"
+            data-max-file-size-mb="5"
```

Comment:
5mb is just an arbitrary number I chose. I think we'll probably want to bump this even higher. Any suggestions? 20mb maybe?

Reply:
Let's use 25mb [that's what I set in the model](https://github.com/thepracticaldev/dev.to/blob/master/app/uploaders/base_uploader.rb#L17)